test/services/authorization-codes.test.js

```diff
@@ -35,7 +27,11 @@ describe('Authorization Code Tests', function () {
       should.exist(code.id);
       code.id.length.should.be.greaterThan(15);
       should.ok(new Date(code.expiresAt) > Date.now());
-      _.omit(code, [ 'id', 'createdAt', 'expiresAt' ]).should.deepEqual(newCode);
+      let filteredCode = Object.assign({}, code);
+      delete filteredCode.id;
+      delete filteredCode.createdAt;
+      delete filteredCode.expiresAt;
+      filteredCode.should.deepEqual(newCode);
```

Comment:
If I understand correctly, the aim of this test is to make sure some properties and values are contained the returned object and not really to be deep equal
We can probably use [should.have.properties](http://shouldjs.github.io/#assertion-properties) instead of removing the properties by hand.

Reply:
👍